### ISSUE TITLE
Bind for peek-legit-discard-file conflicts with vi-mode

### ISSUE THREAD

**Issue by ghosty141** [lem-project/lem#1513]
Because of this keybind you can't navigate in normal mode in legit which makes it kind of unusable.

```(define-key *peek-legit-keymap* "k" 'peek-legit-discard-file)```

**Comment by g-gundam** [comment]
I've encountered quite a few modes where vi-mode feels like a second class citizen.
- color-theme-selector-mode
- dashboard-mode
- legit

Who knows what else.  They make you go into vi insert-mode to do anything.

**Comment by vindarel** [comment]
yeah… those issues are not fun. My "fix" would be to ensure we enter Legit in Emacs mode or vi insert mode, so we don't have the conflicts.

**Comment by g-gundam** [comment]
There has to be a better way.  If Emacs can do it, why not Lem?  I'm going to keep thinking about it.

**Comment by ghosty141** [comment]
> yeah… those issues are not fun. My "fix" would be to ensure we enter Legit in Emacs mode or vi insert mode, so we don't have the conflicts.

I don't fully agree. For VI users normal mode/motion mode is the standard so that should be honored even in 3rd party packages.

For "discard" I'd suggest "x" as that would work with vi-mode in normal just fine! Magit also uses that I believe

**Comment by g-gundam** [comment]
Take a look at #1515.  I didn't do anything for legit yet, but I could.  I just chose to fix some of the simpler modes first as a proof of concept.
